.gitignore

```diff
@@ -56,3 +56,7 @@ docs/_build/
 # PyBuilder
 target/
 .pytest_cache
+
+#vscode extensions
+.vscode
+
```

Comment:
please, delete this file

Reply:
.gitignore file was present before also when I git cloned it.I just added my .vscode extension to it..should I delete the .vscode extension or the whole .gitignore